app/controllers/search/agendaitems.js

```diff
@@ -40,17 +41,21 @@ export default class AgendaitemsSearchController extends Controller {
   @tracked sort;
   @tracked types;
   @tracked latestOnly; // Only show the most recent version of an agenda-item
-  @tracked emptySearch;
+  @tracked searchText;
 
   constructor() {
     super(...arguments);
     this.page = 0;
     this.size = this.sizeOptions[2];
-    this.sort = this.sortOptions[0].value;
```

Comment:
1 of the search test was failing (relevance) because the default sorting is different.
Instead of sorting by most recent by default we are now always sorting on relevance.
Need to adapt the test to this

Reply:
fixed it